frontend/static/js/pages/lobbys/lobby.js

```diff
@@ -231,10 +231,14 @@ var app = new Vue({
         toggleDropdown(id) {
             document.getElementById(id).classList.toggle("active-dropdown")
         },
-
-        async hideLobby(){
+        async hideLobby(event) {
```

Comment:
If you want to separate this into a separate PR we can merge this stuff in right away.